src/clients/KubectlClient.ts

```diff
@@ -130,7 +192,7 @@ export class KubectlClient implements IClient {
     }
 
     public async getIngressesAsync(namespace: string, kubeconfigPath: string, quiet: boolean): Promise<IKubernetesIngress[]> {
```

Comment:
What's the reason why we can be sure the caller will never pass `null` for namespace here, but they might for `getServicesAsync`?

Reply:
@peterbom this is not relevant to this PR. Can we take this separate in the refactoring PR ? Thoughts please ?

Reply:
The reason I thought it was relevant was:
- it'd be good to keep the public API consistent (i.e. always have methods that take a non-null namespace)
- are we sure the signature for `getServicesAsync` (which is part of this PR) is correct? As far as I can tell, the methods are always called with the namespace argument specified, so I'm wondering why we need to deal with possible null values.

Reply:
I agree, you point is valid. , but these are formatting/linting changes which caused these differences.